### DISCUSSION TITLE
How to use EasyOCR in Jetson Nano?

### DISCUSSION THREAD

**Discussion by cheolhun9714** [JaidedAI/EasyOCR#588] [Q&A]
When I use EasyOCR in my laptop, I found my laptop's GPU is not applied for CUDA. So I bought Jetson Nano for using CUDA, but the Jetson Nano make warning "CUDA not available - defaulting to CPU. Note: This module is much faster with a GPU." too.  I googled about "Jetson Nano EasyOCR", and there were some people like me. But the answer in that site was not a solution. How can I solve this?

**Comment by scottauchmoody** [comment]
It sounds like you need to install the correct version of pytorch.
Also, if you've already installed the CPU version of pytorch, you will need to uninstall it.
pip uninstall pytorch torchvision torchaudio
and install the GPU version
pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu117 --no-cache-dir
Note: If you don't include the --no-cache-dir, it will reload the locally cached CPU version if that's already installed.
Also, note: if you do "pip install torch torchvision torchaudio", you will get the CPU only version.
Please check pytorch.org for the latest install information depending on your platform.
The install command should match your installed version of CUDA, version 11.7 in this example.
run a small python program to validate the install.

validate_cuda.py
import torch

if __name__ == '__main__':
    print('torch.cuda.is_available()=', torch.cuda.is_available())
    print('torch.version.cuda=', torch.version.cuda)
    print('cuda.device_count()=', torch.cuda.device_count())
    print('torch version=', torch.__version__)

I get the following(version 11.6 of CUDA)
torch.cuda.is_available()= True
torch.version.cuda= 11.6
cuda.device_count()= 1
torch version= 1.13.1+cu116
